examples/book.rb

```diff
@@ -20,12 +7,11 @@ def initialize(title, author, pages)
     @pages = pages
   end
 
-  def isAlongBook
+  #check if a book is long or not
+  def longBook?
     if @pages >= 100
       return true
     end
```

Comment:
It is preferable to write a single line when the logic inside the if is short. This allows us to reduce the number of lines and keeps the readability of the code. Something like:

```ruby
return true if @pages >= 100
```